Whiskey/Functions/Invoke-WhiskeyTask.ps1

```diff
@@ -244,11 +253,20 @@ function Invoke-WhiskeyTask
     $result = 'FAILED'
     try
     {
+        $TaskContext.Temp = Join-Path -Path $TaskContext.OutputDirectory -ChildPath ('Temp.{0}.{1}' -f $task.Name,[IO.Path]::GetRandomFileName())

```

Comment:
Here you used `$task.Name` but above on line 53 you used `$Name`. Should probably pick one to be consistent.

Reply:
Fixed.